### PR TITLE
added blog on service Mesh Specifications

### LINKED ISSUE #2179: [Blog] Service Mesh Specifications
Contributors and community members are encouraged to post on https://layer5.io/blog. A new blog post fitting the following description is requested to be written by a community member:

**Desired Title:** Service Mesh Specifications and Why They Matter
**Topic:** a write up of a previous KubeCon talk
**Draft:** https://docs.google.com/document/d/1t1mtVUB8_ibUZf73-KHWVJDau8twEj10MCXDPfHk1MY/edit

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabh1908@gmail.com>

**Description**

This PR fixes #2179 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f2975497b0c770332b5b99f1f0b548e578bb8748 at: https://61485275228cbc58d5ac37f3--layer5ng.netlify.app

File: src/collections/blog/2021-09-20-service-mesh-specifications-and-why-they-matter/index.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  <p>
+   <Link to="/community/members/lee-calcote">Lee Calcote</Link> is an innovator, product and technology leader, active in the community as a Docker Captain, Cloud Native Ambassador and GSoC, GSoD, and Community Bridge Mentor. In this talk, he walked through service mesh specifications and why they matter in your deployment.
+   How many service mesh specifications do you know? He went through all of them. So, no worries if you're unfamiliar.
+  </p>
+</div>   
+
+### Service Mesh Specifications:
+<img src={Abstractions} className="slides-left" align="left" alt="al-abstractions" />
+
+As the ubiquity of service eshes unfolds and they become commonplace to any cloud native or edge environment, so does the need for vendor and technology-agnostic interfaces to interact with them. The Service Mesh Interface (SMI), the Service Mesh Performance (SMP), and Multi-Vendor Service Mesh Interoperation (Hamlet) are three open specifications solving the challenge of interoperability, workload and performance management between service meshes.
```
Comment (warunicorn19) [inline src/collections/blog/2021-09-20-service-mesh-specifications-and-why-they-matter/index.mdx:41]:
```suggestion
As the ubiquity of service meshes unfolds and they become a commonplace for any cloud native or edge environment, so does the need for vendor and technology-agnostic interfaces to interact with them. The Service Mesh Interface (SMI), the Service Mesh Performance (SMP), and Multi-Vendor Service Mesh Interoperation (Hamlet) are three open specifications solving the challenge of interoperability, workload and performance management between service meshes. 
```

File: src/collections/blog/2021-09-20-service-mesh-specifications-and-why-they-matter/index.mdx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+There are a number of service meshes out there. One of the community projects is to track the landscape of all of the meshes there are. There’s a lot to say about each of them, their architecture, and their working. Why are they made? Who are they focused on? What do they do? When did they come about? Why are some of them not here anymore? Why are we still seeing new ones? A lot of things to go through. You might be interested in any number of the details that the landscape tracks.
+ <div class="note">Be Aware, It's Meshy Out There!</div>
+
+### Service Mesh Interface
+<img src={SMI} className="slides-left" align="left" alt="smi-image" />
+<ul>
+    <li>It's goal and its genesis were born inside of Kubernetes.</li>
```
Comment (warunicorn19) [inline src/collections/blog/2021-09-20-service-mesh-specifications-and-why-they-matter/index.mdx:59]:
```suggestion
    <li>Its goal and genesis were born inside of Kubernetes.</li>
```
consider using this instead.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by warunicorn19** [comment]
Just minor changing everything else looks good.

**Comment by l5io** [comment]
🚀 Preview for commit 7b7b5bd979d5d43f4058ecb4229003731a6c318c at: https://6148cca6227ea7009266abe3--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 66667b65facae82dc583dc2af77bebddfbc261cc at: https://6148d0e273dec300e83dc27a--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
LGTM🎉, Thanks @debo19.

**Comment by leecalcote** [comment]
Both of you did a fantastic job here. Seeing how much of this you were able to accomplish and maintain high quality with so little review... Wow.

**Pull request merged**